post-content.js

```diff
@@ -26,8 +26,6 @@ window._wpGutenbergPost.content = {
 		'<p style="text-align:right;">... like this one, which is right aligned.</p>',
 		'<!-- /wp:core/text -->',
 
-		'<!-- wp:core/more /-->',
-
```

Comment:
Same comment as above, I think this should use the old-style `<!--more-->` format.